ui/plugins/failure_summary/main.html

```diff
@@ -86,7 +86,7 @@
             <span>is available. This panel will automatically recheck every 5 seconds.</span>
           </div>
         </li>
-        <li ng-if="!tabs.failureSummary.is_loading && !job_log_urls.length">
+        <li ng-if="!tabs.failureSummary.is_loading && !jobLogsAllParsed">
```

Comment:
Testing the PR locally I notice the Failure Summary appears blank on running, pending and coalesced jobs, where on production you would get a "No logs available for this job".

![failuresummaryemptyrunning](https://cloud.githubusercontent.com/assets/3660661/10175260/e83a08c6-66bb-11e5-9586-82b85966ad24.jpg)